meetings/2021-05-26.md

```diff
@@ -0,0 +1,67 @@
+# 2021-05-19 Solid Authorization
+
+https://meet.jit.si/solid-authorization
+
+
+## Agenda
+
+* Minutes https://github.com/solid/authorization-panel/pull/214
+* Use Case 1: Resource Access https://github.com/solid/authorization-panel/pull/215
+* Use Case 3: Inheritance https://github.com/solid/authorization-panel/pull/216
+
+### If time allows (from previous week)
+* Inheritance with [Referenced Data Grants](https://solid.github.io/data-interoperability-panel/specification/#datamodel-referenced-data-grant) (interop panel)
+  * UC [Conditional access by relationship](https://solid.github.io/authorization-panel/authorization-ucr/#conditional-relationship)
+  * Can we 'search and replace' Report -> Project and Measurement -> Task ? This will align it with scenario used in interop spec, primer and shape trees spec
```

Comment:
```suggestion
  * Can we 'search and replace' `Report` -> `Project` and `Measurement` -> `Task`? This will align it with scenario used in interop spec, primer, and shape trees spec
```